s3/s3.go

```diff
@@ -23,13 +23,15 @@ var dynamoSvc *dynamodb.DynamoDB
 var bucket string
 var dynamoTable string
 var keyPrefix string
+var fileExtension string
 
 // Setup sets up AWS S3 connection
 func Setup(c *config.Config) {
 	sess := session.Must(session.NewSession())
 	svc = s3.New(sess, &aws.Config{})
 	bucket = c.S3.Bucket
 	keyPrefix = c.S3.KeyPrefix
+	fileExtension = fmt.Sprintf(".%s", c.S3.FileExtension)
```

Comment:
How about including the period in the string directly?

Reply:
This makes sense. Fixed and squashed.